packages/fxa-settings/src/components/UnitRowWithAvatar/index.test.tsx

```diff
@@ -3,48 +3,29 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import UnitRowWithAvatar from '.';
 
-afterEach(cleanup);
-
 describe('UnitRowWithAvatar', () => {
   it('renders as expected with the default avatar', () => {
-    const { getByTestId, queryByTestId } = render(
-      <UnitRowWithAvatar avatarUrl={null} />
-    );
+    render(<UnitRowWithAvatar avatarUrl={null} />);
 
-    expect(getByTestId('unit-row-with-avatar-route').textContent).toContain(
-      'Add'
-    );
-    expect(getByTestId('unit-row-with-avatar-default')).toHaveAttribute(
-      'role',
-      'img'
-    );
-    expect(getByTestId('unit-row-with-avatar-default')).toHaveAttribute(
-      'aria-label',
-      'Default avatar'
-    );
-    expect(queryByTestId('unit-row-with-avatar-nondefault')).toBeNull();
+    expect(
+      screen.getByTestId('unit-row-with-avatar-route').textContent
+    ).toContain('Add');
+    expect(screen.getByTestId('avatar-default')).toBeInTheDocument;
+    expect(screen.queryByTestId('avatar-nondefault')).toBeNull();
   });
 
   it('renders as expected with the user avatar', () => {
-    const { getByTestId, queryByTestId } = render(
-      <UnitRowWithAvatar avatarUrl="some-fake-image.png" />
-    );
+    render(<UnitRowWithAvatar avatarUrl="some-fake-image.png" />);
```

Comment:
worth asserting the URL? idk

Reply:
Hmm since it's just `#` now maybe not, but when this gets changed to `<Link` later, yes.